wp-content/plugins/wporg-learn/inc/post-meta.php

```diff
@@ -296,15 +296,23 @@ function get_available_post_type_locales( $meta_key, $post_type, $post_status, $
 		$and_post_status = "AND posts.post_status = '$post_status'";
 	}
 
+	$and_post_type = '';
```

Comment:
This allows post type to be removed from the query so that a language applied to any public post type will match. This is necessary for taxonomy filters which are not bound to a particular post type.